src/components/Thanks/SingleVersion/BadgeMilestone.vue

```diff
@@ -106,13 +108,15 @@ const showEqualityBadge = computed(() => props.isGuest || props.onlyKivaCardsAnd
 
 const showBadgeModule = computed(() => showEqualityBadge.value || !!props.badgeAchievedIds.length);
 
+const loansCount = computed(() => props.loans?.length ?? 0);
+
 const moduleTitle = computed(() => {
 	let title = '';
 	if (props.isOptedIn && !props.kivaCardsModuleShown) {
 		title += 'Thank you!<br />';
 	}
 
-	if (!badgeDataAchieved.value?.length) {
+	if (!loansCount.value) {
 		title += 'Take the next step on your impact journey.';
 	} else {
 		title += badgeDataAchieved.value?.length === 1
```

Comment:
Should we also check for `badgeDataAchieved` array lenght?

Reply:
Probably smt like `else if (!badgeDataAchieved.value?.length)` ...

Reply:
Makes sense, just considering a case having loans with no badges, should we show Equality?
I'm confused with those uses cases...

Reply:
We are rendering this component if there are badges only, so we are safe with this condition, thanks checking!

Reply:
If there are loans but no badges we should `JourneyGeneralPrompt`